package-lock.json

```diff
@@ -2246,7 +2246,7 @@
         "import-fresh": "^3.0.0",
         "imurmurhash": "^0.1.4",
         "inquirer": "^6.2.2",
-        "js-yaml": "^3.12.0",
+        "js-yaml": "^3.13.0",
         "json-stable-stringify-without-jsonify": "^1.0.1",
         "levn": "^0.3.0",
         "lodash": "^4.17.11",
```

Comment:
The remediation section of https://snyk.io/vuln/SNYK-JS-LODASH-450202 suggests upgrading to `4.17.12` or higher.

There are a few deps in `package-lock` that still refer to versions below `4.17.12`. Do you anticipate this will cause issues?

Reply:
The carrot `^` will match the latest minor version. So this dep `^4.17.11` can resolve to a higher dep (it depends on the library). This is sufficient to relieve the vuln warnings we are seeing on the broker